src/test/remote/test_remote.py

```diff
@@ -0,0 +1,54 @@
+import yaml
+from subprocess import run, PIPE
+from unittest import TestCase
+from getpass import getuser
+from typing import Dict
+
+from ja.common.proxy.command_handler import CommandHandler
+
+
+class CommandHandlerDummy(CommandHandler):
```

Comment:
This is more like a question, isn't it better to have on CommandHandlerDummy, or do you thing it is better to test the way you did.

Reply:
I assume the intention is to test the real code of the CommandHandler as well

Reply:
@nikolatzotchev I don't understand the question.

Reply:
I was confused because you had 2 different commandHandlerDummys, I think I get it now